data/transactions/logic/assembler_test.go

```diff
@@ -1457,7 +1458,7 @@ done:`
 	require.Equal(t, expectedProgBytes, ops.Program)
 }
 
-func TestMultipleErrors(t *testing.T) {
+func TestSeveralErrors(t *testing.T) {
```

Comment:
Renaming as the natural name for a new unit test to test `func(ops *OpStream) MultipleErrors` is this one.